implementations/lips/head/scheme-banner

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -eu
+lips --version
+exec lips "$@"
```

Comment:
Does `lips` not print a banner by itself?

Reply:
Indeed it does. Fixes coming soon.